book/website/community-handbook/translation/workflow.md

```diff
@@ -0,0 +1,79 @@
+
+#### Current workflow
+
+The current workflow for all languages is using Crowdin from a fork that is temporarily located under a GitHub organization specifically created to host translated versions of _The Turing Way_. 
+This fork is kept updated with the original repository and Crowdin performs automatic machine translations that need review and approval before being accepted as definite versions.
+The deployment of these translated versions is work in progress and some parts of this workflow may change in the future, but for now, feel free to contact anyone on the `#translation` channel of the _The Turing Way_ Slack.
+
+
+#### Define norms of translation
+
+It is key to set translation norms for each language. These norms will guide newcomers on how to translate and review in the translation platform. For instance, we strongly suggest setting a [glossary] through the localisation platform. 
+
+We also suggest settting language-specific guidelines to use inclusive language.
+For example, the book [Teaching Tech Together](https://teachtogether.tech/) was translated to Spanish in a collaborative way and they defined their guidelines to account for regional differences and to avoid gendered and other non-inclusive expressions.
+Their original guidelines (in Spanish) can be found [here](https://github.com/gvwilson/teachtogether.tech/blob/master/es/README.md) and an adaptation of these guidelines to English can be found [here](https://github.com/gvwilson/teachtogether.tech#translations). 
+Document the norms of translation for your selected language to be discussed and updated by the rest of the team.
+
+
+#### Create a dedicated Slack channel
+
+We recommend you create a dedicated Slack channel for the language you are translating within the [_The Turing Way_ slack](theturingway.slack.com). This space will allow you to bring chat with interested parties in your own language. 
+
+#### Organise co-working calls
+
+We suggest defining co-working calls to progress on the translation. These sessions should be open to anyone interested in participating.
+
+You can also work on the translation in the co-working calls of The Turing Way project. You can find this activity in [_The Turing Way_ calendar](https://calendar.google.com/calendar?cid=dGhldHVyaW5nd2F5QGdtYWlsLmNvbQ). Please note that these calls are in English.
+
+#### Asynchronous translation
+
+Everyone interested in participating in a translation effort should be able to work asynchronously, have this in mind when writing the translation norms and any other documentation for your language.
+
+### During translation
+
+#### Before you start translating
+
+Localisation platforms integrate different tools and formats for assisting the translation process. For instance, [Transifex](https://www.transifex.com/) includes the target text blocks to be translated on the left and a translation box on the right. You can select with the cursor which block to translate. 
```

Comment:
Maybe we should remove the specific references to Transifex? It can be confusing given that we are using Crowdin.